Src/Private/Get-AbrFgtSystem.ps1

```diff
@@ -250,43 +250,204 @@ function Get-AbrFgtSystem {
+                            }
 
-                    $OutObj | Table @TableParams
+                            # VLAN interfaces
+                            if ($interfaceType -eq "vlan") {
+                                $vlanUpCount = ($OutObj | Where-Object { $_.Status -eq 'up' }).Count
+                                $vlanDownCount = ($OutObj | Where-Object { $_.Status -ne 'up' }).Count
+                                Paragraph "Total number of unique VLANs found: $($vlanUpCount + $vlanDownCount), of which $vlanUpCount are up and $vlanDownCount are down."
+                                if ($vlanUpCount -gt 0) {
+                                    $vlanUpIDs = ($OutObj | Where-Object { $_.Status -eq 'up' } | Select-Object -ExpandProperty 'VLAN ID' -Unique)
+                                    Paragraph "- Up VLANs are: $($vlanUpIDs -join ', ')."
+                                }
+                                if ($vlanDownCount -gt 0) {
+                                    $vlanDownIDs = ($OutObj | Where-Object { $_.Status -ne 'up' } | Select-Object -ExpandProperty 'VLAN ID' -Unique)
+                                    Paragraph "- Down VLANs are: $($vlanDownIDs -join ', ')."
+                                }
+                                BlankLine
+                            }
+
+                            $downInterfaces = @()
+                            $upInterfaces = @()
+
+                            foreach ($interface in $OutObj) {
+                                if ($interface.PSObject.Properties.Name -contains 'Status') {
+                                    if ($interface.Status -eq 'up') {
+                                        $upInterfaces += $interface
+                                    } else {
+                                        $downInterfaces += $interface
+                                    }
+                                } else {
+                                    $downInterfaces += $interface
+                                }
+                            }
+
+                            if ($upInterfaces.Count -gt 0) {
+                                Write-FormattedTable -InputObject $upInterfaces -TableName $tableName -CustomColumnWidths @{"Name" = 15;"VLAN ID" = 8;"Status" = 10;"IP Address" = 18;"Secondary IP" = 18;"Role" = 8;"Parent Interface" = 12}
+                                $TableParams = @{
+                                    Name         = "Interface"
+                                    List         = $false
+                                    ColumnWidths = 12, 20, 7, 11, 6, 8, 20, 8, 8
+                                }
+
+                                if ($Report.ShowTableCaptions) {
+                                    $TableParams['Caption'] = "- $($TableParams.Name)"
+                                }
+
+                                $OutObj | Table @TableParams
+                            }
+
+                            if ($downInterfaces.Count -gt 0) {
```

Comment:
add option/settings for display not down interface ?

Reply:
When audit the config sometimes, there is some old interface and can be help to have info (an option is good idea for me !)

Reply:
forget to push ?